### PR TITLE
Trigger syntax highlight package release

### PR DESCRIPTION
**Pull request by ksittner**

### CODE REVIEW

File: packages/malloy-syntax-highlight/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "malloy-syntax-highlight",
+  "name": "@malloy-data/syntax-highlight",
```
Comment (whscullin) [inline packages/malloy-syntax-highlight/package.json:2]: Should be @malloydata

Comment (ksittner) [reply packages/malloy-syntax-highlight/package.json]: Fixed

File: packages/malloy-syntax-highlight/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "malloy-syntax-highlight",
+  "name": "@malloydata/syntax-highlight",
   "version": "0.0.82",
-  "description": "An unpublished package to simplify the process of developing and testing Malloy syntax highlighting grammars",
+  "description": "A package to simplify the process of developing, testing, and syncnig Malloy syntax highlighting grammars",
```
Comment (whscullin) [inline packages/malloy-syntax-highlight/package.json:4]: "syncing"

**Review by whscullin — COMMENTED** [review]
I don't know why it's failing, perhaps if you substitute `npm ci --loglevel error` for `npm ci --silent` that might tell us something.

**Review by whscullin — APPROVED** [review]

**Pull request merged**